Monika After Story/game/script-topics.rpy

```diff
@@ -3587,6 +3587,35 @@ label monika_chloroform:
     m 1tsb "You're so cute when you're nervous."
     return
 
+init 5 python:
+    if os.path.isfile(os.path.expandvars("%APPDATA%") + '\RenPy\JustYuri\persistent'):
+      addEvent(Event(persistent.event_database,eventlabel="just_yuri",category=['club members'],prompt="Just Yuri mod",random=True))
```

Comment:
I don't think a feature such as this should be placed in the topics file and shouldn't be repeatable, seeing the nature of the topic. Something like this should probably go in story-events similar to mas_monikai_detected.